torch/optim/rmsprop.py

```diff
@@ -88,7 +88,7 @@ def step(self, closure=None):
                 if group['centered']:
                     grad_avg = state['grad_avg']
                     grad_avg.mul_(alpha).add_(1 - alpha, grad)
-                    avg = square_avg.addcmul(-1, grad_avg, grad_avg).sqrt().add_(group['eps'])
+                    avg = square_avg.addcmul(-1, grad_avg, grad_avg).sqrt_().add_(group['eps'])
                 else:
                     avg = square_avg.sqrt().add_(group['eps'])
```

Comment:
We can use `sqrt_` here too, no?

Reply:
No. Since `square_avg` is a state that we maintain, we should not modify it here.

Reply:
Right, good point.